docs/develop/module-specifications/spec-capability.mdx

```diff
@@ -53,11 +53,11 @@ func NewApp(...) *App {
 
 ## Concepts
 
-Capabilities are multi-owner. A scoped subkeeper can create a capability via `NewCapability,` which creates a unique, unforgeable, object-capability reference. The newly created capability is automatically persisted. The calling module does not need to call `ClaimCapability`. Calling `NewCapability` creates the capability with the calling module and name as a tuple to be the capabilities first owner.
+Capabilities are multi-owner. A scoped subkeeper can create a capability via `NewCapability,` which creates a unique, unforgeable, object-capability reference. The newly created capability is automatically persisted. The calling module does not need to call `ClaimCapability`. Calling `NewCapability` creates the capability with the calling module and name as a tuple to be the capability's first owner.
 
 Capabilities can be claimed by other modules, which add them as owners. `ClaimCapability` allows a module to claim a capability key that it has received from another module so that `GetCapability` calls made later will succeed. If a module that receives a capability wants to access it by name later, `ClaimCapability` must be called. Because capabilities are multi-owner, if multiple modules have one capability reference, all the modules own it. If a module receives a capability from another module but does not call `ClaimCapability`, it may use it in the executing transaction but will not be able to access it afterward.
 
-Any module can call `AuthenticateCapability` to check whether a capability corresponds to a particular name, including un-trusted user input, with which the calling module previously associated it.
+Any module can call `AuthenticateCapability` to check whether a capability corresponds to a particular name, including un-trusted user input, which the calling module previously associated with it.
```

Comment:
How can we simplify this? 

We should turn this phrase into 2 separate sentences.

Reply:
@evanorti I think "Any module can call `AuthenticateCapability` to check whether a capability corresponds to the name which the calling module previously associated with it." is what they meant?